src/core/forms.py

```diff
@@ -619,3 +620,27 @@ def order_by(self, queryset, facet, fks):
         # function property like journal.name
 
         return queryset
+
+
+class EmailForm(forms.Form):
+    cc = forms.CharField(
+        required=False,
+        max_length=1000,
+        help_text='Separate email addresses with ;',
+    )
+    subject = forms.CharField(max_length=1000)
+    body = forms.CharField(widget=SummernoteWidget)
+
+    def clean_cc(self):
+        cc = self.cleaned_data['cc']
+        if not cc or cc == '':
```

Comment:
The second operand here is redundant, the expression should shortcircuit on the first conditional when `cc` is set to `''` (thought I see this code was copied verbatim from `journal/views.py`)